src/sider/arbeidsuforhet/ForhandsvarselSendt.tsx

```diff
@@ -1,50 +1,23 @@
 import React from "react";
-import { Alert, Box, Button, Heading } from "@navikt/ds-react";
-import { ButtonRow } from "@/components/Layout";
-import { useArbeidsuforhetVarselDocument } from "@/hooks/arbeidsuforhet/useArbeidsuforhetVarselDocument";
-import { VisBrev } from "@/components/VisBrev";
-
-const texts = {
-  title: "Venter på svar fra bruker",
-  iSendt: "Forhåndsvarselet er sendt til bruker",
-  isPassert: "Tiden har gått ut på forhåndsvarselet.",
-  sendtInfo:
-    "Om du får svar fra bruker, og hen oppfyller kravene om 8-4 etter din vurdering, klikker du på “oppfylt”-knappen under. Om ikke må du vente til tiden går ut før du kan gi avslag.",
-  passertInfo: "Tiden har gått ut og du kan nå gå videre med å sende avslag.",
-  seSendtBrev: "Se sendt brev",
-  oppfylt: "Oppfylt",
-  avslag: "Avslag",
-  frist: "Fristen går ut:",
-};
+import { useArbeidsuforhetVurderingQuery } from "@/data/arbeidsuforhet/arbeidsuforhetQueryHooks";
+import { addWeeks } from "@/utils/datoUtils";
+import { ForhandsvarselBeforeDeadline } from "@/sider/arbeidsuforhet/ForhandsvarselBeforeDeadline";
+import { ForhandsvarselAfterDeadline } from "@/sider/arbeidsuforhet/ForhandsvarseAfterDeadline";
+import dayjs from "dayjs";
 
 export const ForhandsvarselSendt = () => {
-  const { getForhandsvarselDocument } = useArbeidsuforhetVarselDocument();
+  const { data } = useArbeidsuforhetVurderingQuery();
+  const forhandsvarsel = data[0];
+  const frist = addWeeks(new Date(forhandsvarsel.createdAt), 3);
```

Comment:
Denne burde vi vel egentlig få fra backend (svarfrist-felt på varselet tilknyttet forhåndsvarsel-vurderingen) i stedet for at vi må ha logikk for antall uker her også.

Reply:
Så da tenker du at dette ikke kan merges før den endringen er gjort?

Reply:
Neida, kan godt merge denne og så fikse det etterpå